docs/content/en/docs/reference/clusterspec/nutanix.md

```diff
@@ -0,0 +1,322 @@
+
+### workerNodeGroupConfigurations.count
+Number of worker nodes. Optional if `autoscalingConfiguration` is used, in which case count will default to `autoscalingConfiguration.minCount`.
+
+### workerNodeGroupConfigurations.machineGroupRef (required)
+Refers to the Kubernetes object with Nutanix specific configuration for your nodes. See `NutanixMachineConfig` fields below.
+
+### workerNodeGroupConfigurations.name (required)
+Name of the worker node group (default: `md-0`)
+
+### workerNodeGroupConfigurations.autoscalingConfiguration.minCount
+Minimum number of nodes for this node group’s autoscaling configuration.
+
+### workerNodeGroupConfigurations.autoscalingConfiguration.maxCount
+Maximum number of nodes for this node group’s autoscaling configuration.
+
+### workerNodeGroupConfigurations.taints
+A list of taints to apply to the nodes in the worker node group.
+ 
+Modifying the taints associated with a worker node group configuration will cause new nodes to be rolled-out, replacing the existing nodes associated with the configuration.
+ 
+At least one node group must not have `NoSchedule` or `NoExecute` taints applied to it.
+
+### workerNodeGroupConfigurations.labels
+A list of labels to apply to the nodes in the worker node group. This is in addition to the labels that EKS Anywhere will add by default.
+ 
+Modifying the labels associated with a worker node group configuration will cause new nodes to be rolled out, replacing the existing nodes associated with the configuration.
+
+### datacenterRef
+Refers to the Kubernetes object with Nutanix environment specific configuration. See `NutanixDatacenterConfig` fields below.
+
+### kubernetesVersion (required)
+The Kubernetes version you want to use for your cluster. Supported values: `1.23`, `1.22`, `1.21`, `1.20`
+
+## NutanixDatacenterConfig Fields
+
+### endpoint (required)
+The Prism Central server fully qualified domain name or IP address. If the server IP is used, the PC SSL certificate must have an IP SAN configured.
+
+### port (required) 
+The Prism Central server port. (Default: `9443`) 
+
+### insecure (optional)
+Set insecure to `true` if the Prism Central server does not have a valid certificate. (Default: `false`)
+
+### additionalTrustBundle (optional; required if using a self-signed PC SSL certificate)
+The PEM encoded CA trust bundle.
+
+The `additionalTrustBundle` needs to be populated with the PEM-encoded x509 certificate of the Root CA that issued the certificate for Prism Central. Suggestions on how to obtain this certificate are [here]({{< relref "../nutanix/nutanix-prereq/#prepare-a-nutanix-environment" >}}).
+
+## NutanixMachineConfig Fields
+
+### cluster
+Reference to the Prism Element cluster.
+
+### cluster.type	
+Type to identify the Prism Element cluster. (Permitted values: `name` or `uuid`)
+ 
+### cluster.name	
+Name or UUID of the Prism Element cluster.
```

Comment:
This can only be the name. We need another entry for cluster.uuid
```suggestion
Name of the Prism Element cluster.

### cluster.uuid
UUID of the Prism Element cluster.
```